openedx/openedx-platform
# [BD-29] TNL-7260 LearningSequences Course Outline makes sequences with hide_after_due inaccessible after due date

LearningSequences Course Outline makes sequences with hide_after_due inaccessible after due date

**JIRA tickets**: 
[OSPR-4801](https://openedx.atlassian.net/browse/OSPR-4801)
[TNL-7260](https://openedx.atlassian.net/browse/TNL-7260)

**Dependencies**: None

**Sandbox URL**: TBD - sandbox is being provisioned.

**Testing instructions**:

1. Pull the `open-craft:patrick/BB-2672-tnl-7260-course-outline-api-hide-after-due` branch locally and setup your local LMS.
2. Modify a unit in the Studio to have a due date in the past
3. Run `make lms-shell`
4. Run `python manage.py lms migrate learning_sequences`
5. Run `python manage.py lms update_course_outline <COURSE_KEY>`
6. Navigate to 
```
http://localhost:18000/api/learning_sequences/v1/course_outline/<COURSE_KEY>?user=verified
```
7. Look through the response for the unit that was marked with a due date in the past. It should have the `accessible` attribute set to `false`

**Author notes and concerns**:

**Reviewers**
